app/src/main/java/lt/getpet/getpet/adapters/OnboardingAdapter.kt

```diff
@@ -0,0 +1,67 @@
+package lt.getpet.getpet.adapters
+
+import android.content.Context
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentPagerAdapter
+import lt.getpet.getpet.constants.ActivityConstants.Companion.POSITION
+import lt.getpet.getpet.fragments.OnBoardingFragment
+
+
+/**
+ * Created by Greta Grigutė on 2018-11-10.
+ */
+class OnboardingAdapter(private val mContext: Context, fm: FragmentManager) : FragmentPagerAdapter(fm) {
+
+    private var mBundle: Bundle? = null
+    private var mFragment: Fragment? = null
+    private var mPosition: Int? = null
+
+    override fun getItem(position: Int): Fragment? {
+        mPosition = position
+        when (position) {
+            0 -> {
+                mBundle = Bundle()
+                mBundle!!.putInt(POSITION, 0)
+                mFragment = OnBoardingFragment()
```

Comment:
Create `newInstance` method with parameter position to `OnBoardingFragment` and set bundle inside `newInstance`. Example: https://github.com/GotPet/getpet-android/blob/29c463b868a604cea375440d436f6e371670c4e5/app/src/main/java/lt/getpet/getpet/fragments/UserProfileFragment.kt#L82

Reply:
In this case you won't even need to use `when`:
```kotlin
override fun getItem(position: Int): Fragment {
    return OnBoardingFragment.newInstance(position)
 }
```

Reply:
Changed OnboardingAdaper code, very nice suggestion :)